environments/_users/swiss.yml

```diff
@@ -3,6 +3,7 @@ dev_users:
     - ddobre
     - pdelcroix
```

Comment:
This looks like the same person, FYI. Maybe just change their keys?

Reply:
Yeah. That looks right. Thanks for pointing it out. Would update the PR.

Reply:
The keys for the user already exists. Closing this PR.